frontend/src/components/eMIB/transformations.js

```diff
@@ -0,0 +1,28 @@
+// transform addressBook (list of contactShapes) into a list of display names (optionShapes)
```

Comment:
Is it possible to put this file under the **helper** folder, as there are already the same type of files in there?

Reply:
moved in 593b0a0